### PR TITLE
Presentation: Initial API for property formatting on frontend

### PR DESCRIPTION
**Pull request by saskliutas**
Investigated APIs available on frontend for quantity values formatting and added basic formatter that can format property values in Content.

Part of https://github.com/iTwin/itwinjs-core/issues/5029

### CODE REVIEW

File: presentation/common/src/presentation-common/PropertyFormatter.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import { Format, FormatProps, FormatterSpec, ParserSpec, UnitsProvider } from "@itwin/core-quantity";
+import { Content } from "./content/Content";
+import { Field, PropertiesField } from "./content/Fields";
+import { DisplayValue, NestedContentValue, Value } from "./content/Value";
+import { KindOfQuantityInfo, PropertyInfo } from "./EC";
+import { ValuesDictionary } from "./Utils";
+
+/** @alpha */
+export interface FormatOptions {
+  name: string;
```
Comment (grigasp) [inline presentation/common/src/presentation-common/PropertyFormatter.ts:19]: not clear whose name this is - maybe rename to `koqName` or something like that?

**Review by grigasp — APPROVED** [review]

File: presentation/common/src/presentation-common/PropertyFormatter.ts (hunk 1 above)
Comment (saskliutas) [reply presentation/common/src/presentation-common/PropertyFormatter.ts:19]: This is the name of `Format` and `FormatterSpec` not `KOQ`. Both requires name to be provided so I just decided to used name of `KOQ` for that.

File: presentation/common/src/presentation-common/PropertyFormatter.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  formatProps: FormatProps;
+  persistenceUnitName: string;
+}
+
+/** @alpha */
+export class PropertyFormatter {
```
[the remark this answers is not in the document]
Comment (saskliutas) [reply presentation/common/src/presentation-common/PropertyFormatter.ts]: Changed

File: presentation/common/src/presentation-common/PropertyFormatter.ts (hunk 1 above)
Comment (saskliutas) [reply presentation/common/src/presentation-common/PropertyFormatter.ts]: Removed name as it is not used by `Format` or `FormatterSpec`

**Review by grigasp — APPROVED** [review]

**Pull request merged**